--- bin/user/belchertown.py.orig	2025-08-11 00:05:44.907933691 +0200
+++ bin/user/belchertown.py	2025-08-11 00:06:21.739276942 +0200
@@ -1672,17 +1672,17 @@
                 if self.generator.skin_dict["Extras"]["earthquake_server"] == "USGS":
                     eqtime = eqdata["features"][0]["properties"]["time"] / 1000
                     equrl = eqdata["features"][0]["properties"]["url"]
                     if distance_unit == "km":
                         eqplace = eqdata["features"][0]["properties"]["place"]
                     else:  # assume miles
                         try:
                             eqmatched = match(
-                                "(?P<distance>[0-9]*\\.?[0-9]+) km(?P<rest>.*)$",
+                                r"(?P<distance>[0-9]*\.?[0-9]+) km(?P<rest>.*)$",
                                 eqdata["features"][0]["properties"]["place"],
                             )
                             eqdist_km = eqmatched.group("distance")
                             eqdist_miles = round(float(eqdist_km) / 1.609, 1)
                             eqplace = (
                                 str(eqdist_miles) + " miles" + eqmatched.group("rest")
                             )
                         except: